packages/wme-ui/src/components/jump-nav/jump-nav.tsx

```diff
@@ -7,9 +7,9 @@ import { styled } from '@mui/material/styles';
 interface JumpNavProps extends BoxProps {
   title?: string;
   links: Array<{
-    id: number;
+    id: string;
     label: string;
-    href: string;
+    onClick: () => void;
```

Comment:
I'm not sure if replacing the `href` prop with `onClick` is the right thing to do here. As I commented above, we should get clarification on the functionality of the JumpNav links.